emu_mps/mps_backend.py

```diff
@@ -66,3 +77,26 @@ def _run(impl: MPSBackendImpl) -> Results:
             os.remove(impl.autosave_file)
 
         return impl.results
+
+
+def permute_bitstrings(perm: torch.Tensor, results: Results) -> None:
+    if "bitstrings" not in results.get_result_tags():
+        return
+    uuid_bs = results._find_uuid("bitstrings")
+
+    results._results[uuid_bs] = [
+        Counter({optimat.permute_string(bstr, perm): c for bstr, c in bs_counter.items()})
+        for bs_counter in results._results[uuid_bs]
+    ]
+
+
+def permute_correlations(perm: torch.Tensor, results: Results) -> None:
```

Comment:
the naming is misleading, a minute ago, I wrote a comment that you forgot to permute the occupation result, yet here it is.

Reply:
My point was that it's a one- and two-point correlation functions. I'll give a better name then